src/interface/obsidian/src/chat_view.ts

```diff
@@ -76,6 +77,10 @@ export class KhojChatView extends KhojPaneView {
 
         // Clear text after extracting message to send
         let user_message = input_el.value.trim();
+		// Store the message in the array if it's not empty
+		if (user_message) {
+			this.userMessages.push(user_message);
```

Comment:
We should also preload the `this.userMessages` list with the values of the user messages retrieved in `getChatHistory`. You can do it in the `chatLogs.forEach` loop. Check if the message was sent by the user, and if so, push it into the array. Otherwise looks good!